src/components/Modal/Modal.js

```diff
@@ -339,18 +348,22 @@ export default class Modal extends Component {
     );
 
     return (
-      <div
-        {...other}
-        onKeyDown={this.handleKeyDown}
-        onClick={this.handleClick}
-        onBlur={this.handleBlur}
-        className={modalClasses}
-        role="presentation"
-        tabIndex={-1}
-        onTransitionEnd={this.props.open ? this.handleTransitionEnd : undefined}
-        ref={this.outerModal}>
-        {modalBody}
-      </div>
+      <FocusTrap active={focusTrap}>
```

Comment:
Looks that `active` here is for activating focus-wrap behavior. When I close the modal by changing `open` knob at https://deploy-preview-2096--carbon-components-react.netlify.com/experimental/?selectedKind=Modal&selectedStory=Default and manually put e.g. `<input type="text">` into `<body>` by DOM inspector, I can never focus on that `<input>` - That said, did we want:

```suggestion
      <FocusTrap active={open && focusTrap}>
```

?